app/src/main/res/layout/activity_profile.xml

```diff
@@ -68,17 +113,31 @@
 
     <EditText
         android:id="@+id/editTextUsername"
-        android:imeOptions="actionDone"
-        android:layout_height="70dp"
+        style="@style/ButtonGrey"
         android:layout_width="300dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
+        android:layout_height="70dp"
+        android:ems="10"
+        android:hint="@string/insert_username_button"
+        android:imeOptions="actionDone"
+        android:inputType="textPersonName"
         app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
```

Comment:
Cirrus complains that autofillHints is missing. Would be nice if that was added, even with some null solutions.

Reply:
Added autofillHints!